flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -731,70 +697,6 @@ public void invoke() throws Exception {
 		}
 	}
 
-	@Test
-	public void testTaskInterruptionAndTerminationOnShutdown() throws Exception {
```

Comment:
By removing this test we lose coverage that we don't stop the `TaskExecutor` before all tasks have been terminated. I couldn't find another test testing this scenario. Hence, please add one or revert the removal.

Reply:
I returned the test but refactored it to use `StubTaskSlotTable` and checking waiting for `TaskSlotTable.closeAsync` while terminating `TaskExecutor`. Interruption of tasks in `TaskSlotTable.closeAsync` is tested separately.